server.js

```diff
@@ -1,11 +1,40 @@
+const fs = require('fs')
 const express = require("express");
-const app = express();
 
+// Add this to the top of your file
+const exphbs = require("express-handlebars");
+const app = express();
+// Then these two lines after you initialise your express app
+app.engine("handlebars", exphbs({ defaultLayout: "main" }));
+app.set("view engine", "handlebars");
 // The extensions 'html' allows us to serve file without adding .html at the end 
 // i.e /my-cv will server /my-cv.html
-app.use(express.static("public", {'extensions': ['html']}));
-
+app.get("/", (req, res) => {
+  const filePath = __dirname + "/data/posts.json";
+  const callbackFunction = (error, file) => {
+    // we call .toString() to turn the file buffer to a String
+    const fileData = file.toString();
+    // we use JSON.parse to get an object out the String
+    const postsJson = JSON.parse(fileData);
+    // send the json to the Template to render
+    res.render("index", {
+      title: "Raj Profile", // insert your name instead
+      posts: postsJson
+    });
+  };
+  fs.readFile(filePath, callbackFunction);
+});
 
+app.get("/admin", (req, res) => {
+  res.render("admin", {title: "Welcome to Admin"});
+});
+app.get("/my-cv", (req, res) => {
+  res.render("my-cv", { title: "Repos List From Github" });
+});
+app.get("/contact", (req, res) => {
+  res.render("contact", {title: "Welcome to Contact"});
+});
+app.use(express.static("public", { 'extensions': ['html'] }));
 // what does this line mean: process.env.PORT || 3000
```

Comment:
If you don't understand something, the first place you should look to is Google! :D Take a look at https://www.twilio.com/blog/2017/08/working-with-environment-variables-in-node-js.html to understand Node.js environment variables. 

Regarding the || This is the way of setting a default value if the first value is null. So in this case, if `process.env.PORT` is null, it will default to 3000

Reply:
Yes, I did not answer here! Thanks for the clear explanation..